### ISSUE TITLE
Large fonts cause labels to go out of borders

### ISSUE THREAD

**Issue by Shushandr** [RaJiska/BeyondPaste#6]
![изображение](https://github.com/RaJiska/BeyondPaste/assets/148656348/a39eaf40-077b-46ac-a15f-90ae06397f9c)

**Comment by RaJiska** [comment]
Yes, it seems to happen when the window is resized too small or when going through mobile. I'm afraid I am not good enough in frontend to make a responsive design :) I'll just leave it as such as I can't be bothered with all this design stuff
